spec/lib/container_orchestrator_spec.rb

```diff
@@ -82,6 +82,62 @@
         )
       end
     end
+
+    it "sets database environment variables" do
+      stub_const("ENV", ENV.to_h.merge(
+        "DATABASE_HOSTNAME" => "postgres",
+        "DATABASE_NAME"     => "vmdb_production",
+        "DATABASE_PASSWORD" => "password",
+        "DATABASE_PORT"     => "5432",
+        "DATABASE_REGION"   => "0",
+        "DATABASE_SSL_MODE" => "verify-full",
+        "DATABASE_USER"     => "postgres",
+      ))
+
+      expect(subject.send(:default_environment)).to include(
+        {:name => "DATABASE_PORT",     :value => "5432"},
+        {:name => "DATABASE_SSL_MODE", :value => "verify-full"},
+        {:name => "DATABASE_HOSTNAME", :valueFrom => {:secretKeyRef => {:key => "hostname", :name => "postgresql-secrets"}}},
+        {:name => "DATABASE_NAME",     :valueFrom => {:secretKeyRef => {:key => "dbname",   :name => "postgresql-secrets"}}},
+        {:name => "DATABASE_PASSWORD", :valueFrom => {:secretKeyRef => {:key => "password", :name => "postgresql-secrets"}}},
+        {:name => "DATABASE_USER",     :valueFrom => {:secretKeyRef => {:key => "username", :name => "postgresql-secrets"}}},
+      )
+    end
+  end
+
+  context "#deployment_definition (private)" do
+    it "skips the database root certificate if the orchestrator doesn't have it" do
+      expect(File).to receive(:file?).with("/.postgresql/root.crt").and_return(false)
+      allow(File).to receive(:file?).and_call_original
```

Comment:
This is a bit confusing: why the double `expect`/`allow` for `File.exist?`?  Shouldn't we just do the `expect`, and let any other callers raise an exception?

If there is another `File.exist?` call that this is covering, I think you should comment on what that is here for future devs.

Reply:
There is at least one other call (loading the GUID) that fails without the `allow`

Reply:
Ah, okay, that makes sense.  I sting think then just doing:

```ruby
allow(File).to receive(:file?).and_call_original # allows other calls to .file? to still work
```

Would make this more obvious, but I am not terribly insistent on that.

Reply:
Done